src/components/explorer/ExplorerSearchResultsTable.js

```diff
@@ -83,6 +86,13 @@ const ExplorerSearchResultsTable = ({ data, activeTab, ...props }) => {
         ],
     };
 
+    const sortedInfo = useMemo(() => {
+        return {
+            order: props.sortOrder,
+            columnKey: props.sortColumnKey,
+        };
+    }, [props.sortOrder, props.sortColumnKey]);
```

Comment:
odd mixed style here (all other props in this component are destructured, ie called by `someName`, rather than `props.someName`).. although in general the repo overall has wildly varying styles...

Reply:
Good catch! thanks